virttest/qemu_devices/qcontainer.py

```diff
@@ -639,14 +639,19 @@ def simple_unplug(self, device, monitor):
         # Remove all devices, which are removed together with this dev
         out = device.unplug(monitor)
 
-        ver_out = device.verify_unplug(out, monitor)
-
-        if ver_out is False:
+        from virttest import utils_misc
+        if not utils_misc.wait_for(
+                lambda: device.verify_unplug(out, monitor) is True, timeout=20):
```

Comment:
I see, yes this makes sense. Would you please add a comment. Also you claim it takes 5-7s on your machines so 20s seems quite tight (imagine low-cost arm/riscv machines). How about 60s or at least 30s?

Reply:
... besides it should finish earlier unless unplug it's broken...